ci/build-docker-image.sh

```diff
@@ -42,74 +41,147 @@ do
         --ssh )
             CI_DOCKER_SSH=true ;;
 
+        --ref-name=* )
+            CI_REF_NAME="${i#*=}" ;;
+
+        --platforms=* )
+            CI_DOCKER_PLATFORMS="${i#*=}" ;;
+
+        --push_image=* )
+            CI_DOCKER_PUSH_IMAGE="${i#*=}" ;;
```

Comment:
Do you want me to add comments about them? Because these are making the script more generic, multi-arch and compatible with multiple Git hosting solutions

Reply:
It needs to be documented somewhere, whether this is in this script or somewhere else doesn't matter for me.